gebru/roster.json

```diff
@@ -17,6 +17,10 @@
     {
       "name": "Trina Yau",
       "github": "trinayau"
+},
+    {
+      "name": "Prishal Makwana",
+      "github": "Prishal-git"
```

Comment:
Be careful, no to modify too much :) 
Can you please make sure that your curly braces are aligned @Prishal-git ?

Reply:
@rom-30 
Thanks for the feedback - I have amended it and have done another pull request. I hope this was the correct method to rectify previous pull requests